go.mod

```diff
@@ -322,6 +321,7 @@ replace (
 	k8s.io/kms => k8s.io/kms v0.26.11
 	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.26.11
 	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.26.11
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20230501164219-8b0f38b5fd1f
```

Comment:
this needs to be pinned because new version from `k8s.io/api` does not compatible with current version. Reference: https://github.com/kubernetes/kube-openapi/issues/404